toolbox/githubctl/main.go

```diff
@@ -303,6 +277,39 @@ func DailyReleaseQualification(baseBranch *string) error {
 	return fmt.Errorf("release qualification failed")
 }
 
+func traverseJobTree(job config.Postsubmit, postsubmitJobs *[]string, targetBranch string) {
+	for _, branch := range job.Brancher.Branches {
+		if branch == targetBranch {
+			*postsubmitJobs = append(*postsubmitJobs, job.Name)
+			if len(job.RunAfterSuccess) > 0 {
+				for _, childJob := range job.RunAfterSuccess {
+					traverseJobTree(childJob, postsubmitJobs, targetBranch)
+				}
+			}
+			break
+		}
+	}
+}
+
+func readPostsubmitListFromProwConfig(org, repo, branch string) []string {
+	var postsubmitJobs []string
+	repoRootDir, err := u.Shell("git rev-parse --show-toplevel")
+	repoRootDir = strings.TrimSuffix(repoRootDir, "\n")
+	if err != nil {
+		glog.Fatalf("cannot find repo root directory path")
+	}
+	prowConfigYaml := filepath.Join(repoRootDir, "prow/config.yaml")
```

Comment:
I think in golang you can give relative path. Just add it as a const at the top of the file.

Reply:
Yes relative path works but I think absolute path is better because it is less susceptible to directory restructuring. If this go file is moved elsewhere it would still work.